.github/workflows/update-gh-pages.yml

```diff
@@ -10,6 +10,147 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
+    - name: Install Python
+      uses: actions/setup-python@v5
+      with:
+        python-version: '3.13'
```

Comment:
Let's make this an environment variable and declare it at the top of the file, under the `env` table. This way, we will be able to easily spot variables used in this file.

Reply:
Agreed - also recommended to use 3.12 for now.